content/guides/comparison.md

```diff
@@ -46,3 +51,10 @@ webpack is not the only module bundler out there. If you are choosing between us
 ♦ in production mode (opposite in development mode)
 
 X is the length of the path string
+
+
+## Bundling vs. Loading
+
+It's important to note some key differences between _loading_ and _bundling_ modules. A tool like [SystemJS](https://github.com/systemjs/systemjs), which can be found under the hood of [JSPM](http://jspm.io/), is used to load and transpile modules at runtime in the browser. This differs significantly from webpack, where modules are transpiled (through "loaders") and bundled before hitting the browser.
+
+Each method has it's pros and cons. Loading and transpiling modules at runtime can add a lot of overhead for larger sites and applications comprised of many modules. For this reason, SystemJS makes more sense for smaller projects where fewer modules are required. However, this may change as load times decrease and [HTTP/2](https://http2.github.io/) gains support.
```

Comment:
Maybe you can differentiate between transfer from server and compiling/processing on client more clearly. HTTP/2 only improves the former. The latter is especially important for mobile devices.